cmd/util.go

```diff
@@ -121,6 +115,24 @@ func PersistentPreRunEnv(cmd *cobra.Command, nodeType node.Type, _ []string) err
 	gateway.ParseFlags(cmd, &cfg.Gateway)
 	state.ParseFlags(cmd, &cfg.State)
 
+	switch nodeType {
+	case node.Light:
+		err = header.ParseFlags(cmd, &cfg.Header)
+		if err != nil {
+			return err
+		}
```

Comment:
@walldiss I didn't add it because light node does not support "pruning" right now, so it will be added once it is supported.

Reply:
If Light node does not support Prunner, perhaps pruner and pruning service should not be provided to FX for Light nodes.

Reply:
Done.

Reply:
```
	// TODO: Eventually, light nodes will be capable of pruning samples
	//  in which case, this can be enabled.
	case node.Light:
		return fx.Module("prune",
			fx.Supply(light.Window),
		)
```